workflows/sarek.nf

```diff
@@ -294,17 +253,32 @@ workflow SAREK {
     // To gather used softwares versions for MultiQC
     versions = Channel.empty()
 
-    // Download cache if needed
-    // Assuming that if the cache is provided, the user has already downloaded it
-    ensemblvep_info = params.vep_cache    ? [] : Channel.of([ [ id:"${params.vep_cache_version}_${params.vep_genome}" ], params.vep_genome, params.vep_species, params.vep_cache_version ])
-    snpeff_info     = params.snpeff_cache ? [] : Channel.of([ [ id:"${params.snpeff_genome}.${params.snpeff_db}" ], params.snpeff_genome, params.snpeff_db ])
-
+    // Download cache
     if (params.download_cache) {
-        PREPARE_CACHE(ensemblvep_info, snpeff_info)
-        snpeff_cache = PREPARE_CACHE.out.snpeff_cache
-        vep_cache    = PREPARE_CACHE.out.ensemblvep_cache.map{ meta, cache -> [ cache ] }
-
-        versions = versions.mix(PREPARE_CACHE.out.versions)
+        // Assuming that even if the cache is provided, if the user specify download_cache, sarek will download the cache
```

Comment:
I would stop that pipeline with a message telling the user that the combination of options don't make sense.

Reply:
That's what we were doing, and what's causing the issue. Given that these params have default, I think it's easier for the user to just use download_cache without having to disable 2 params if the need arise